### DISCUSSION TITLE
[Bug]: Uncaught ReferenceError: quicklink is not defined

### DISCUSSION THREAD

**Discussion by ixbarbarbar** [GetPublii/Publii#1754] [Bug Report]
### Operating system

Debian bookworm (6.1.106-3)

### Publii version

0.46.1 (build 16918)

### Post editor

None

### Bug description

When browsing the site's pages, the following error appears in the logs and in the web development tool

`Loading failed for the <script> with source “./media/plugins/pageprefetching/quicklink.umd.js”`
`....`
`Uncaught ReferenceError: quicklink is not defined`

It's due to the issue to load the library quicklink.umd.js who is in the folder media => plugins => pagePrefetching but in the generated html the pagePrefetching is without the P in uppercase

`<script src="./media/plugins/pageprefetching/quicklink.umd.js">`

Additional informations : 
* HTTP web server : Nginx
* Theme : Square version 3.0.0.0

### Steps to reproduce

1. Install the version 0.46.1 of publii (but exist in previous version)
2. Install Square theme version 3.0.0.0
3. Generate the site
4. Launch the preview or navigate to your website

**Comment by bobmitro** [comment]
Thanks for reporting. The bug has been fixed in plugin version 1.1.0 
Link: https://marketplace.getpublii.com/plugins/page-prefetching/

**Comment by ixbarbarbar** [comment]
Thanks

Oups sorry, I forgot that I have install this plugin in addition and not check if there is a new version.
